app/src/main/java/com/moviefiend/torian/moviefiend/MovieDetailsFragment.java

```diff
@@ -0,0 +1,60 @@
+package com.moviefiend.torian.moviefiend;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.moviefiend.torian.moviefiend.network.NowPlayingResponse;
+import com.squareup.picasso.Picasso;
+
+public class MovieDetailsFragment extends Fragment {
+
+    public static final int MOVIE_POSTER_SIZE = 780;
+
+    private NowPlayingResponse.MovieInfo mMovieInfo;
+    private ImageView mPosterView;
+    private TextView mRatingView;
+    private TextView mDescriptionView;
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.movie_details_fragment, container, false);
+
+        mMovieInfo = getArguments().getParcelable("movie_info");
+
+        mPosterView = (ImageView) view.findViewById(R.id.poster);
+        mRatingView = (TextView) view.findViewById(R.id.rating);
+        mDescriptionView = (TextView) view.findViewById(R.id.description);
+
+        setTitle(mMovieInfo.getTitle());
+        setPosterView(getString(R.string.tmdb_poster_url, MOVIE_POSTER_SIZE, mMovieInfo.getPosterPath()));
```

Comment:
It's better to build URLs using `Uri.Builder` instead of strings manipulation.
Specifically here you're not modifying the URLs with user input but to prevent any malicious attempts and to ensure proper encoding it's better to use `Uri.Builder`.